data/Nigeria/Representatives/sources/instructions.json

```diff
@@ -37,6 +37,21 @@
         "from": "election-wikidata",
         "base": "Q22283753"
       }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT * FROM data WHERE position = 'Federal Representative'"
+      },
+      "source": "http://www.shineyoureye.org/media_root/popolo_json/persons.json",
```

Comment:
This field doesn't really need to be a full URL like that — it would probably be better just as 'http://www.shineyoureye.org'